options.c

```diff
@@ -3255,6 +3255,8 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 		.help	= "Your platform does not support IO scheduler switching",
 	},
 #endif
+
+	/* Parameters for zones defined in the fio job */
```

Comment:
Maybe these should be queued in another commit?

Reply:
I'll take them out. I think the comments give helpful structure, but they certainly aren't related to any priority changes.